x-pack/plugins/apm/public/components/shared/charts/transaction_charts/mobile_agent_transaction_charts.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGrid, EuiFlexItem, EuiPanel } from '@elastic/eui';
+import React from 'react';
+import { AnnotationsContextProvider } from '../../../../context/annotations/annotations_context';
+import { ChartPointerEventContextProvider } from '../../../../context/chart_pointer_event/chart_pointer_event_context';
+import { ServiceOverviewThroughputChart } from '../../../app/service_overview/service_overview_throughput_chart';
+import { LatencyChart } from '../latency_chart';
+import { FailedTransactionRateChart } from '../failed_transaction_rate_chart';
+
+export function MobileAgentTransactionCharts({
```

Comment:
I think we could skip the `Agent` in the name IMHO `MobileTransactionCharts` is good.

Reply:
I agree, removed it in https://github.com/elastic/kibana/pull/143697/commits/9cab05301c4352db61467fd0923f573308443f48